docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -159,6 +159,20 @@ See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getA
 <2> Use the assertion library provided by the Synthetics agent to look for the
 expected value. See https://jestjs.io/docs/expect[Jest expect docs] for more information.
 
+[discrete]
+[[synthetics-create-test-script-recorder]]
+=== Synthetics Recorder
+
+beta[]
+
+Use the Synthetics Recorder to generate code for one or more steps by interacting with a web page directly.
```

Comment:
I've included some comments with thoughts about the questions posed in these comments.